elastic/kibana
# [A11Y][SLO detail] burn rate time range abbreviations

Fixes https://github.com/elastic/observability-accessibility/issues/54

## How to test
- Turn on your Screen Reader (voiceOver on MAC)
- Go to SLO detail page
- The burn rate abbreviation labels should be `1 hour`, `6 hours`, `24 hours`, `72 hours`
- Navigate with `Tab` button to the burn rate time range abbreviations
- The Screen Reader should speak out `1 hour`, `6 hours`, `24 hours`, `72 hours`



https://github.com/user-attachments/assets/4cc6a375-afef-4531-a177-d52e9326a701
